src/Microsoft.DotNet.ProjectModel/ProjectReaderSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.ProjectModel
+{
+    public class ProjectReaderSettings
+    {
+        public string VersionSuffix = null;
```

Comment:
I'm not aware of any other cases where we expose fields like this. Propert would match what we do everywhere else.